examples/js/objects/BillboardMesh.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * @author Max Strater, Autodesk, Inc. / http://www.autodesk.com/
+ *
+ * A billboard mesh that faces a specified point, optionally constrained to an axis
+ * Based on http://nehe.gamedev.net/article/billboarding_how_to/18011/
+ */
+
+// expects a normalized axis
+THREE.BillboardMesh = function ( geometry, material, point, normalizedAxis, constrainedToAxis ) {
+
```

Comment:
Yes, An axis in three.js is always assumed to be normalized. You can call the parameter `axis`, and keep your comment for clarity -- as we have done elsewhere in the library.